testsuite/MDAnalysisTests/utils/test_modelling.py

```diff
@@ -35,13 +36,10 @@
 )
 import MDAnalysis.core.groups
 from MDAnalysis.core.groups import AtomGroup
-from MDAnalysisTests import parser_not_found, tempdir
 
-from numpy.testing import (TestCase, dec, assert_equal, assert_raises, assert_,
+from numpy.testing import (assert_equal, assert_raises, assert_,
```

Comment:
Did you remove all  `assert_raises` and `assert_` from this file. Then they shouldn't be imported anymore